test/Project.toml

```diff
@@ -24,9 +24,9 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 [preferences.OrdinaryDiffEq]
```

Comment:
When changing the values here, maybe also update
https://github.com/trixi-framework/Trixi.jl/blob/main/docs/src/troubleshooting.md#installing-and-updating-everything-takes-a-lot-of-time
?

Reply:
I would not like to do this. The part of our docs is meant for interactive use. The settings shown there will precompile also methods used by default in Trixi.jl. Thus, it pays off to precompile them since they will be used several times and I assume that people do not update OrdinaryDiffEq.jl every time they restart Julia. However, this is different in our CI setups where we need to pay the cost of precompilation for every run without any chance of amortization.

Reply:
Good point! /r